### PR TITLE
Error Handling

### PR DESCRIPTION
**Pull request by raufabr**
@danielgrijalva thought I would create an early PR to get some feedback. 
This wasn't part of the original bug, if you search Fifa 19 you would get an error previously and I've added a check here.

Will fix the other ones as I keep going.

### CODE REVIEW

**Comment by danielgrijalva** [comment]
Thanks @raufabr, looks good so far - just be careful not to remove the `.env.default` files 😅

**Comment by raufabr** [comment]
@danielgrijalva I've added the final checks in the details tab. 

Spitan Spirits Expedition - in the Phantasmagoria wasn't rendering for 3 problems
1) It didn't have any screenshots -  I've modified the code to show the cover for the time being.
2) There were no countries associated with the developers - added a rendering check for this
3) There were no involved companies - added the rendering check.

File: frontend/src/modules/game/index.js [hunk 1]
```diff
@@ -59,6 +59,7 @@ class Game extends React.Component {
   //loads the new game
   loadGame = gameSlug => {
     axios.get(`/api/games/${gameSlug}`).then(res => {
+      console.log(res.data[0]);
```
Comment (danielgrijalva) [inline frontend/src/modules/game/index.js]: Console logs should be removed! 😄

**Comment by danielgrijalva** [comment]
Regarding your fix to 1., if the game doesn't have any screenshots, then the backdrop component should not be displayed at all, not even with the cover. Can you update it?

Also, there already seems to be a fix for the countries and companies bugs you pointed out (#101). Apologies for not letting you now 😅 you're gonna have to undo these changes.

File: frontend/src/modules/game/index.js (hunk 1 above)
Comment (raufabr) [reply frontend/src/modules/game/index.js]: My bad. Easy fix this one.

**Comment by raufabr** [comment]
My bad, I'll update the changes. 
Out of curiosity, why are we not just showing the game cover if there are no screenshots?

**Comment by danielgrijalva** [comment]
@raufabr mainly because the aspect ratio of game covers is different from most screenshots/artwork and they would look stretched if used in the backdrop, which I find kinda ugly... so I'd rather not to have anything in there!

Example:
> image

**Comment by raufabr** [comment]
@danielgrijalva  one more question. Should we be mapping all the developers, or should we showing the first one?

`  getDeveloperName = companies => {
    var dev = companies.find(c => {
      return c.developer === true || {};
    });

    return dev.company.name;
  };`

**Comment by danielgrijalva** [comment]
@raufabr I don't remember if right now we are showing _all_ of them or just the first one... seems to me that it would be more informative/complete to show all the developers and publishers involved, right?

Would you be interested in adding the code for showing all the companies involved? (in a separate PR)

**Comment by raufabr** [comment]
I'll pick this up. Do you want me to create a feature request for this too?

**Comment by danielgrijalva** [comment]
Nah, that's not necessary!

**Comment by danielgrijalva** [comment]
@raufabr it seems that we're already showing multiple developers...

> image

**Comment by raufabr** [comment]
@danielgrijalva I've pushed all the changes as you suggested. This should be fixed now. :grinning:
If everything looks good, I can pick up new features in the front end/testing front end for the weekend 

:muscle:
--

**Comment by danielgrijalva** [comment]
@raufabr great! thanks, we do need help with front end tests now that the suite is merged into master, CI and code coverage :sweat_smile:

**Pull request merged**